Meta/Lagom/Tools/CodeGenerators/LibWeb/BindingsGenerator/IDLGenerators.cpp

```diff
@@ -1920,6 +1921,15 @@ JS_DEFINE_NATIVE_FUNCTION(@class_name@::@function.name:snakecase@@overload_suffi
     [[maybe_unused]] auto& realm = *vm.current_realm();
 )~~~");
 
+    if (function.extended_attributes.contains("FIXME")) {
+        function_generator.append(R"~~~(
+            dbgln("FIXME: Unimplemented IDL interface '@namespaced_name@.@function.name@'");
+            return JS::js_undefined();
```

Comment:
This is super-nitpicky, but the alignment of the generated code is slightly off here. It should be aligned with the generated code above.